src/lucky/cookies/cookie_jar.cr

```diff
@@ -0,0 +1,56 @@
+class Lucky::CookieJar
+  alias Key = String | Symbol
+  private property store : HTTP::Cookies
+  @changed = false
+  EXPIRATION      = 1.year.from_now
```

Comment:
@Sija made a good point that this should be a method (or value) so if a crystal app is running for a long time, the value will be updated.

If you compile this gist you can see that the constant stays, well, constant while the method updates: https://gist.github.com/edwardloveall/da12432b41704187c1a175dceffe3e7a

Reply:
This should be `1.year` (`Time::Span`).
Also possibly a `class_property` to make it configurable if needed.